projects/Mallard/src/App.tsx

```diff
@@ -131,7 +131,7 @@ const handleIdStatus = (attempt: AnyAttempt<IdentityAuthData>) =>
 
 export default class App extends React.Component<{}, {}> {
     componentDidMount() {
-        SplashScreen.hide()
+        setTimeout(() => SplashScreen.hide(), 1000)
```

Comment:
```suggestion
        setTimeout(() => { SplashScreen.hide() }, 1000)
```

I like to make lambdas not return unless the return value is useful but I have no idea if this has any actual benefit or it's just me being pedantic